addons/payment_custom/models/payment_transaction.py

```diff
@@ -31,6 +31,22 @@ def _get_specific_rendering_values(self, processing_values):
             'reference': self.reference,
         }
 
+    def _get_custom_communication(self):
```

Comment:
To me this communication thing is not closely related to the "custom trait" of the `payment_custom` module; the method could just be:
```suggestion
    def _get_communication(self):
```
or 
```suggestion
    def _select_communication(self):
```